kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,3 +1,14 @@
+local function check_status(status)
+  if not status then
```

Comment:
Yep, see [TravisCI Build #7300.4](https://travis-ci.org/Mashape/kong/jobs/238393754#L1641-L1649) from before adding the null check.

```
[----------] Running tests from spec/03-plugins/23-aws-lambda/01-access_spec.lua
./kong/plugins/aws-lambda/schema.lua:2: attempt to compare number with nil
stack traceback:
	./kong/plugins/aws-lambda/schema.lua:2: in function 'func'
	./kong/dao/schemas_validation.lua:215: in function 'validate_entity'
	./kong/dao/schemas_validation.lua:186: in function 'validate'
	./kong/dao/model_factory.lua:27: in function 'validate'
	./kong/dao/dao.lua:118: in function 'insert'
	spec/03-plugins/23-aws-lambda/01-access_spec.lua:55: in function <spec/03-plugins/23-aws-lambda/01-access_spec.lua:6>
[----------] 0 tests from spec/03-plugins/23-aws-lambda/01-access_spec.lua (736.92 ms total)
```

Pushed an update in 292d9a2404a.